content/master/concepts/pods.md

```diff
@@ -0,0 +1,384 @@
+status of deployed resources and correcting any "drift." After checking a 
+resource Crossplane waits some time and checks again.
+
+Crossplane monitors resources through a Kubernetes 
+_[watch](https://kubernetes.io/docs/reference/using-api/api-concepts/#efficient-detection-of-changes)_
+or through periodic polling. Some resources may be both watched and polled. 
+
+Crossplane requests that Kubernetes notifies Crossplane of any changes on
+objects. This notification tool is a _watch_. 
+
+Watched objects include Providers, managed resources and 
+CompositeResourceDefinitions.
+
+For objects that Kubernetes can't provide a watch for, Crossplane
+periodically poll the resource to find it's state. The default polling rate is
+one minute. Change the polling rate with the `---poll-interval` pod argument.
+
+Reducing the poll-interval value causes Crossplane to poll resources more
+frequently. This increases the load of the Crossplane pod and
+results in more frequent provider API calls.
+
+<!-- vale write-good.TooWordy = NO -->
+<!-- allow "maximum" -->
+Increasing the poll-interval causes Crossplane to poll resources less
+frequently. This increases the maximum time until Crossplane
+discovers changes in the cloud provider that require updating. 
+<!-- vale write-good.TooWordy = YES -->
+
+Managed resources use polling.
+
+{{< hint "note" >}}
+Managed resources watch for Kubernetes events like deletion or changes to
+their `spec`. Managed resources rely on polling to detect changes in the
+provider environment.
+{{< /hint >}}
+
+Crossplane checks all resources to
+confirm they're in the desired state. Crossplane does this every one hour by
+default. Use the `--sync-interval` Crossplane pod argument to change this
+interval. 
+
+The `--max-reconcile-rate` rate defines the rate, in times per second, 
+Crossplane reconciles resources. 
+
+Reducing the `--max-reconcile-rate`, or making it smaller, reduces CPU 
+resources Crossplane uses, but increases the amount of time until changed 
+resources are fully synced. 
+
+Increasing the `--max-reconcile-rate`, or making it larger, increases the
+CPU resources Crossplane uses but allows Crossplane to reconcile all resources
+faster. 
+
+{{< hint "important" >}}
+Most Providers use their own `--max-reconcile-rate`. This determines the
+same settings for Providers and their managed resources. Applying the
+`--max-reconcile-rate` to Crossplane only controls the rate for
+core Crossplane resources. 
+{{< /hint >}}
+##### Reconcile retry rate
+
+The `--max-reconcile-rate` setting configures the number of times per second
+Crossplane or a provider attempts to correct a resource. The default value is 
+10 times per second.
+
+All core Crossplane components share the reconcile rate. Each Provider
+implements their own max reconcile rate setting. 
+
+##### Number of reconcilers
+
+The second value `--max-reconcile-rate` defines is the number of threads
```

Comment:
I'd still prefer not to talk about threads. It's not accurate. Could we find some other approachable way to phrase this? i.e. How many resources can be reconciled concurrently?

Reply:
I'm struggling with this. I understand why you want to avoid `thread` but it seems like we're hinting at something without actually naming it. How is this edit? 
---
The second value `--max-reconcile-rate` defines is the number of
resources that can reconcile at once. If there are more resources than
configured `--max-reconcile-rate` the remaining resources must wait until
Crossplane reconciles a an existing resource.

Read the [Change Pod Settings]({{<ref "#change-pod-settings">}}) section for
instructions on applying these settings.

Reply:
That sounds great to me.